cli/integrity/integrity_test.go

```diff
@@ -0,0 +1,97 @@
+package integrity
```

Comment:
Please, test only the public API.
```suggestion
package integrity_test
```

Reply:
Fixed